docs/_partials/reverification-route-handler.mdx

```diff
@@ -0,0 +1,20 @@
+The following example uses the [`has()`](/docs/references/nextjs/auth#has) helper to check if the user has verified their credentials within a specific time period. The `strict` configuration sets the time period to 10 minutes. If the user hasn't verified their credentials within 10 minutes, the `reverificationError` utility is used to return an error and initiate the session reverification flow on the client-side.
```

Comment:
Inside route handlers devs should use `reverificationErrorResponse` instead of `reverificationError`.

```suggestion
The following example uses the [`has()`](/docs/references/nextjs/auth#has) helper to check if the user has verified their credentials within a specific time period. The `strict` configuration sets the time period to 10 minutes. If the user hasn't verified their credentials within 10 minutes, the `reverificationErrorResponse` utility is used to return an error and initiate the session reverification flow on the client-side.
```